### ISSUE TITLE
ConnectionAbortedError

### ISSUE THREAD

**Issue by susnick** [jay0lee/got-your-back#161]
This error only shows up on this one mailbox.  Let me know what you require to fix this issue.

Got 1107446 Message IDs
GYB needs to examine 1107446 messages
GYB already has a backup of 1097910 messages
GYB needs to backup 9536 messages
Traceback (most recent call last):
  File "gyb.py", line 1580, in <module>
  File "gyb.py", line 1044, in main
  File "gyb.py", line 487, in callGAPI
  File "oauth2client\util.py", line 137, in positional_wrapper
  File "googleapiclient\http.py", line 1397, in execute
  File "googleapiclient\http.py", line 1337, in _execute
  File "oauth2client\transport.py", line 169, in new_request
  File "httplib2\__init__.py", line 1314, in request
  File "httplib2\__init__.py", line 1064, in _request
  File "httplib2\__init__.py", line 988, in _conn_request
  File "http\client.py", line 1106, in request
  File "http\client.py", line 1151, in _send_request
  File "http\client.py", line 1102, in endheaders
  File "http\client.py", line 936, in _send_output
  File "http\client.py", line 908, in send
  File "ssl.py", line 891, in sendall
  File "ssl.py", line 861, in send
  File "ssl.py", line 586, in write
ConnectionAbortedError: [WinError 10053] An established connection was aborted by the software in your host machine
Failed to execute script gyb

**Comment by jay0lee** [comment]
This is typically firewall, security or antivirus software/hardware on your computer or network blocking GYB. Please check there. Closing.

**Comment by BetoHertt** [comment]
I had the same issue with only 2 emails from 60 that are being stored.

After trying a lot of things I found out it was the internet connection. I did a restart at the fiber modem and the problem was over. No need to change the firewall or antivirus for me.

I makes sense now because some computers were having trouble to download files from the internet. They were failing to download too many times even if they were small files like 50MB.

I hope it helps or at least give you new ideas to check.
